test/REQUIRE

```diff
@@ -0,0 +1,8 @@
+julia 0.3
```

Comment:
Hm, good question. It might not be! If we can't find it, or any open issues about it, then we should open a doc to-do issue to add some.